### PR TITLE
0.2/672 styling for diff states

### PR DESCRIPTION
**Pull request by alexalala**
https://trello.com/c/hpghB6ZA/672-styling-for-different-states-basic-accessibility-standards-on-hold-waiting-for-tests-from-charlie

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -88,6 +88,11 @@ input[type="checkbox"]+label:before {
 	cursor: pointer;
 }
 
+input[type="radio"]:focus+label:before,
+input[type="checkbox"]:focus+label:before {
+	box-shadow: 0 0 0 3px #dcb9d2;
```
Comment (charlesstrange2) [inline public/css/main.css:93]: Task card says "blue" but the pinky colour #dcb9d2 works; amend card to make this clear?

File: public/css/main.css [hunk 2]
```diff
@@ -131,14 +135,41 @@ select {
 
 button {
 	background-color: #a74e94;
+	transform: translate3d(0, 0, 0);
```
Comment (charlesstrange2) [inline public/css/main.css:134]: caniusethis.com thinks translate3d should work in this context on ie10/11 but not ie9. The audit isn't done yet, but this should be fine :-/

**Review by charlesstrange2 — APPROVED** [review]
Cool - Just note that we're not sure of the support criteria (unfortunatly) so be conservative with the shiney. 

We may have to come back to this if they're running it on steam powered windows 7!

File: public/css/main.css (hunk 1 above)
Comment (alexalala) [reply public/css/main.css:93]: True!! Thanks.

**Pull request merged**